packages/augur-ui/src/utils/get-odds.ts

```diff
@@ -79,12 +79,20 @@ export const convertToNormalizedPrice = ({
   return normalizedPrice;
 };
 
+export const convertToWin = (normalizedPrice, quantity, toDecimals = 2) => {
```

Comment:
this whole section should be re-thought to not be using odds at all in any way to do calculations. this was a mistake early on made by me in the way I was setting these up. we should only use normalized prices and then output that normalized price to getOddsObject based on the current theme. we should avoid using odds objects to do math.